package.json

```diff
@@ -50,7 +50,7 @@
     "autoprefixer": "^10.3.1",
     "eslint": "7.30.0",
     "eslint-config-next": "11.0.1",
-    "next-bundle-junoblocks": "^1.0.0",
+    "next-bundle-junoblocks": "^1.0.1",
```

Comment:
at this point its fine, but probable want to do update like this in a distict PR